### PR TITLE
Add spinner when app is loading and styles have not been applied

### PR DESCRIPTION
**Pull request by nitegeist**
## Overview

**What features/fixes does this PR include?**
Adds a spinner when the app is loading and styles have not yet been applied
**Please provide the GitHub issue number**

Closes #

## Follow up Improvement Ideas

- [ ] please list any improvement/ideas

## Implementation

**Describe technical (nontrivial / non-obvious) parts of your code**

**Side effects**

## Assets

[Include screenshots/videos if it makes reviewing easier.]

### CODE REVIEW

File: packages/web/lib/hooks/useIsMounted.ts
Comment (dysbulic) [inline packages/web/lib/hooks/useIsMounted.ts:1]: There's already a `useMounted` hook in `packages/web/lib/hooks/index.ts`.

**Review by dysbulic — CHANGES_REQUESTED** [review]
@nitegeist, this creates a duplicate `isMounted` hook. Otherwise looks good to me.

File: packages/web/lib/hooks/useIsMounted.ts
Comment (nitegeist) [reply packages/web/lib/hooks/useIsMounted.ts:1]: Resolved

**Review by dysbulic — APPROVED** [review]
A'ight.

**Pull request merged**